src/pages/AddProperty/AddProp.jsx

```diff
@@ -124,7 +125,7 @@ class AddProp extends React.Component {
                 <TopNav />
                 <div className="saveDraft-publish-buttons">
                     <button>Save As Draft</button>
-                    <button id="pubButton">Publish Listing</button>
+                    <button id="pubButton onClick={this.submitProperty}">Publish Listing</button>
```

Comment:
you have quotes surrounding the function. should be `<button id="pubButton" onClick={this.submitProperty}>` that's why the css wasn't showing up on it. Make that minor change and you are good to go!

Reply:
omg, I can't believe I missed that lol . Hey at least I got some experience with git out of it lol . Thank You!